src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer.cs

```diff
@@ -130,7 +132,8 @@ public sealed partial class DeclarePublicApiAnalyzer : DiagnosticAnalyzer
                     SymbolDisplayParameterOptions.IncludeName |
                     SymbolDisplayParameterOptions.IncludeDefaultValue,
                 miscellaneousOptions:
-                    SymbolDisplayMiscellaneousOptions.UseSpecialTypes);
+                    SymbolDisplayMiscellaneousOptions.UseSpecialTypes |
+                    (SymbolDisplayMiscellaneousOptions)IncludeNullableReferenceTypeModifier);
```

Comment:
❓ What happens if the analyzer passes this flag to an older version of the compiler? If the input is validated to not contain extraneous flags, then we should conditionally include this option similar to the following:

```csharp
private static readonly SymbolDisplayMiscellaneousOptions IncludeNullableReferenceTypeModifier
    = Enum.TryParse(nameof(IncludeNullableReferenceTypeModifier), out SymbolDisplayMiscellaneousOptions value) ? value : SymbolDisplayMiscellaneousOptions.None;
```

Reply:
I reviewed the usage of `MiscellaneousOptions` in the compiler code and I don't think there is any problem. We have a method to add a flag, another to remove a flag, a method to replace the whole set, and one to test the presence of a flag. All of these are safe in the presence of unanticipated flags.